add note for installing the cli extension

nquinlan opened the pull request:
### Why:

While this page is about enabling the CLI extension, as a user it can be confusing to not be able to run `gh copilot` despite it being enabled. Without deeper knowledge of the GitHub CLI and extensions, it's hard to understand that there might be another step to get it working on your machine.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

I added a `{% note %}` callout to your page on enabling GitHub Copilot CLI to highlight for users that they also need to install a CLI extension.

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

---

Based on the [styleguide](https://docs.github.com/en/contributing/style-guide-and-content-model/style-guide#callouts), I'm not sure two notes callouts are the right option here. Based on the styleguide, I'd suggest making the note on permissions simply inline, but I'm not the expert on your docs by any means. Please let me know what you'd like and I can update my PR.

Thanks!

nquinlan committed:
add note for installing the cli extension

nquinlan committed:
fixed broken link

nguyenalex836 commented:
@nquinlan Thanks so much for opening a PR! I'll get this triaged for review ✨

hubwriter commented on content/copilot/github-copilot-in-the-cli/enabling-github-copilot-in-the-cli.md:
```diff
@@ -15,6 +15,12 @@ shortTitle: Enabling Copilot in the CLI
 
 {% endnote %}
 
+{% note %}
+
+**Note:**  After enabling {% data variables.product.prodname_copilot_cli %}, you'll need to install the {% data variables.product.prodname_copilot_cli %} extension, see "[AUTOTITLE](/copilot/github-copilot-in-the-cli/using-github-copilot-in-the-cli)."
+
+{% endnote %}
+
 The {% data variables.product.prodname_copilot_cli %} public beta is available to all organizations{% ifversion ghec %} and enterprises{% endif %} that have an active {% data variables.product.prodname_copilot_for_business %} license. You can enable or disable {% data variables.product.prodname_copilot_cli_short %} for your organization{% ifversion ghec %} or enterprise{% endif %} in the {% data  ...
```
I think it's a great idea to add a note about this, but I think it would be better to place it at the end of this introductory section.

Because this article is aimed at administrators who will enable Copilot in the CLI for people, rather than the people using it themselves, I think it would be better to talk about "users" rather than "you" here. 

So maybe change this to:

```suggestion
The {% data variables.product.prodname_copilot_cli %} public beta is available to all organizations{% ifversion ghec %} and enterprises{% endif %} that have an active {% data variables.product.prodname_copilot_for_business %} license. You can enable or disable {% data variables.product.prodname_copilot_cli_short %} for your organization{% ifversion ghec %} or enterprise{% endif %} in the {% data variables.product.prodname_copilot_short %} settings.

{% note %}

**Note:** To use {% data variables.product.prodname_copilot_cli %}, after it has been enabled, users must install the {% data variables.product.prodname_copilot_cli %} extension, see "[AUTOTITLE](/copilot/github-copilot-in-the-cli/using-github-copilot-in-the-cli)."

{% endnote %}
```

hubwriter reviewed (changes requested):
@nquinlan - Many thanks for raising this PR.

Adding this will definitely improve this article. Thanks for the idea and proposing the change, using all the correct Markdown. Much appreciated.

I've suggested a slight change. If you agree, please apply the change and we'll get this merged.

Thanks again for contributing to our docs. 🙇

nquinlan committed:
move note to the bottom of the introduction, with user reference

> Because this article is aimed at administrators who will enable Copilot in the CLI for people, rather than the people using it themselves, I think it would be better to talk about "users" rather than "you" here.

Co-authored-by: hubwriter <hubwriter@github.com>

nquinlan commented:
Thank you so much! I'm glad it was useful. Committed your suggested update.

Excited to have landed my first change to GitHub Inc docs. 😄

hubwriter committed:
Merge branch 'main' into patch-1

hubwriter reviewed (approved):
Great. Thanks again. 
I'll get this merged/published.

CASABECI reviewed (commented):
Wkn
